src/routes/safe/components/Apps/hooks/useAppList.ts

```diff
@@ -40,12 +45,13 @@ const useAppList = (): UseAppListReturnType => {
         try {
           const currentApp = list[index]
 
-          const appInfo: any = await getAppInfoFromUrl(currentApp.url)
+          const appInfo: SafeApp = await getAppInfoFromUrl(currentApp.url)
           if (appInfo.error) {
             throw Error(`There was a problem trying to load app ${currentApp.url}`)
           }
 
-          appInfo.disabled = currentApp.disabled === undefined ? false : currentApp.disabled
+          appInfo.disabled = Boolean(currentApp.disabled)
+          appInfo.isDeletable = currentApp.isDeletable === undefined ? true : currentApp.isDeletable
```

Comment:
Can we use the same approach with Boolean as above? Exactly the same but with logical negation

Reply:
I think we can't because if it's `undefined`, we need to assign `true`. and if we negate it, it won't work as expected when `currentApp.isDeletable` has a boolean value

Reply:
oh sorry, you're right